### DISCUSSION TITLE
Why is triplet repeated in manifest-generated vcpkg_installed directory tree?

### DISCUSSION THREAD

**Discussion by sullivak-kyra** [microsoft/vcpkg#23175] [Q&A] [answered]
I'm trying out manifests, starting with a simple one lib example
`{
  "$schema": "https://raw.githubusercontent.com/microsoft/vcpkg/master/scripts/vcpkg.schema.json",
  "name": "mxmia",
  "version": "0.0.0",
  "dependencies": [    
    "itk"
  ]
}`
it seems to work (well except this: #20672), but I was surprised to see the vcpkg_installed\x64-windows\ directory has as a sub-directory, another x64-windows, that has all the bin, include etc.
 Is this normal?

**Comment by autoantwort** [comment, marked as answer]
Yes

**Reply by Osyotr** [reply]
Really? My installed path is `vcpkg_installed/x64-windows` and not `vcpkg_installed/x64-windows/x64-windows`.

**Reply by autoantwort** [reply]
It depends. When you use the msbuild integration the second variant is used otherwise the first variant
